website/snippets/_packages_or_dependencies.md

```diff
@@ -0,0 +1,32 @@
+
+## Use cases
+
+Starting from dbt v1.6, `dependencies.yml` has replaced `packages.yml`. The `dependencies.yml` file can now contain both types of dependencies: "package" and "project" dependencies.
+- "Package" dependencies lets you add source code from someone else's dbt project into your own, like a library. 
+- "Project" dependencies provide a different way to build on top of someone else's work in dbt. Refer to [Project dependencies](/docs/collaborate/govern/project-dependencies) for more info.
+- You can rename your existing `packages.yml` to `dependencies.yml` unless you need to use Jinja within your packages specification. This is necessary if you want to add an environment variable with a Git token in a private Git package specification.
+
+There are some important differences between using a `dependencies.yml` compared to a `packages.yml` file:
```

Comment:
The information below gets more at the distinction between `packages` and `projects` (the two ways of taking a dependency on another project), versus the distinction between `packages.yml` and `dependencies.yml`. I think framing it as the difference between the two kinds of files is more confusing.